examples/flask/README.md

```diff
@@ -5,7 +5,7 @@ Simple website made with Flask and the Patreon OAuth API
 ## Install
 
 1. `./install.sh && source venv/bin/activate`
-2. Fill out `config.py` with your client details
+2. Fill out `my_site/config.py` with your client details
```

Comment:
@21echoes Any specific reason for this change?

Reply:
it's the accurate relative path to the `README.md` file -- when i was going to test out my changes, i couldnt quickly find `config.py` until i looked inside `my_site` :x